libchihua-msg/src/node.c

```diff
@@ -0,0 +1,81 @@
+#include "../include/node.h"
+#include "../include/node/peer-dbus.h"
+#include "./common/memory.h"
+
+#include <stdio.h>
+
+Node *node_new(const NodeParams *p) {
+        fprintf(stdout, "Creating Node...\n");
+
+        int r = 0;
+        sd_event *event = NULL;
+        r = sd_event_new(&event);
```

Comment:
Based on https://www.freedesktop.org/software/systemd/man/sd_event_new.html: 
> It is recommended to use this call instead of sd_event_new() in order to share event loop objects between various components that are dispatched in the same thread. All threads have exactly either zero or one default event loop objects associated, but never more.
Since this is part of the library I thought idea that each node should have its own, isolated event loop - independent of the thread it is running in. In our case, both should work fine as the node will be the only thing running in the thread. I'll change it to `sd_event_default()`.